### ISSUE TITLE
How to get gradients from a model?

### ISSUE THREAD

**Issue by sha-hsg** [keras-team/keras#10278]
Hi,

We would like to compute a gradient for a given network parameter set. This might take place in analogy to receiving the loss value with help of `model.set_parameters()` and consequently calling `model.evaluate()`. So far we didn't find any possibility to receive the loss gradient at the point given by `model.set_parameters()`. There doesn't seem to be a way to get the gradient values alone from the model. How can this be done?

Cheers

**Comment by brge17** [comment]
```keras.backend.gradients``` (https://keras.io/backend/)
